src/Service.Tests/GraphQLBuilder/MutationBuilderTests.cs

```diff
@@ -1100,6 +1192,15 @@ string[] expectedNames
                 FieldDefinitionNode createMutation = mutation.Fields.First(f => f.Name.Value == expectedCreateMutationName);
                 Assert.AreEqual(expectedCreateMutationDescription, createMutation.Description.Value);
 
+                if (GraphQLTestHelpers.DoesRelationalDBSupportMultipleCreate(entityNameToDatabaseType[entityNames[i]]))
```

Comment:
why is this check needed when the mutation root is provided with: `IsMultipleCreateOperationSupportedAndEnabled: true)`?

Reply:
Not needed, removed it.